### ISSUE TITLE
Include credentialSubject examples that use URLs rather than DIDs

### ISSUE THREAD

**Issue by selfissued** [w3c/vc-data-model#1309]
It appears that all the VP examples use DIDs as the `credentialSubject` ID (whereas URLs are used for the `issuer`).  Is there a reason for that?  For an example, see https://www.w3.org/TR/vc-data-model-2.0/#example-a-simple-example-of-a-verifiable-presentation.

It would seem logical to add examples where the `credentialSubject` uses a URL rather than a DID (or modify some existing examples to do so).

**Comment by msporny** [comment]
Yes, agreed, we can change some examples to use a URL for the `credentialSubject` identifier.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2024-02-14-vcwg#section2-2) on 2024-02-14

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 2.2. Include credentialSubject examples that use URLs rather than DIDs (issue vc-data-model#1309)

_See github issue [vc-data-model#1309](https://github.com/w3c/vc-data-model/issues/1309)._




**Brent Zundel:** Include credentialSubject examples that use URLs rather than DIDs #1309. some agreement by manu, raised by selfissued. who can be assigned?  

**Gabe Cohen:** I can take it.  

> *Dave Longley:* notably, a DID *is* a URL ... but i presume this means "HTTPS" URLs.

</details>

**Comment by TallTed** [comment]
It occurs to me that it might make sense to also use a DID to identify the `issuer` in at least one example, just to make it clearer that DIDs are just as valid as any other URL as identifier of either or both `credentialSubject` and `issuer`.
